dist/images/daemonset.sh

```diff
@@ -508,20 +513,23 @@ ovn_image=${image} \
   j2 ../templates/ovs-node.yaml.j2 -o ${output_dir}/ovs-node.yaml
 
 # ovn-setup.yaml
-net_cidr=${OVN_NET_CIDR:-"10.128.0.0/14/23"}
-svc_cidr=${OVN_SVC_CIDR:-"172.30.0.0/16"}
+net_cidr=${OVN_NET_CIDR:-"10.244.0.0/16"}
+svc_cidr=${OVN_SVC_CIDR:-"10.96.0.0/16"}
```

Comment:
Hmm why changed the above CIDRs ?

Reply:
After discussing this with @msherif1234, I've just removed the part that was aligning the default values for CIDRs across different scripts. It was not essential for this PR.